CHANGELOG.md

```diff
@@ -29,7 +43,7 @@ https://github.com/openenclave/openenclave/issues/3555 tracks this.
      - Fix TOCTOU vulnerability in NULL terminator checks for ocall in/out string parameters.
      - Count/size properties in deep-copied in/out structs are treated as readonly to prevent the host
 	   from changing corrupting enclave memory by changing these properties.
-
+- Fixed socket syscalls can leak enclave memory contents (CVE-2020-15224).
```

Comment:
I think if the intent is to use the advisory name, then it's clearer if we actually link the advisory here. 
```suggestion
- Fixed [Socket syscalls can leak enclave memory contents](https://github.com/openenclave/openenclave/security/advisories/GHSA-525h-wxcc-f66m) (CVE-2020-15224).
```

Reply:
Fixed.